src/Pos/Wallet/Web/Api.hs

```diff
@@ -68,13 +73,13 @@ import           Pos.Types                  (Coin, SoftwareVersion)
 import           Pos.Util.Servant           (CCapture, CQueryParam, CReqBody,
                                              DCQueryParam, ModifiesApiRes (..),
                                              ReportDecodeError (..), VerbMod)
-import           Pos.Wallet.Web.ClientTypes (Addr, CAccount, CAccountId, CAccountInit, CCoin,
-                                             CAccountMeta, CAddress, CElectronCrashReport,
-                                             CId, CInitialized, CPaperVendWalletRedeem,
-                                             CPassPhrase, CProfile, CTx, CTxId, CTxMeta,
-                                             CUpdateInfo, CWallet, CWalletInit,
-                                             CWalletMeta, CWalletRedeem, SyncProgress,
-                                             Wal)
+import           Pos.Wallet.Web.ClientTypes (Addr, CAccount, CAccountId, CAccountInit,
```

Comment:
@Pastafarianist I'm now using `.stylish-haskell.yaml` from `serokell-util`, but diff remains.
Problem with previous version was in `CCoin` being at wrong place